spring-cloud-generator/generate-all.sh

```diff
@@ -18,6 +20,7 @@ save_error_info () {
 # repos are downloaded once before all generation jobs and then removed
 
 bash download-repos.sh
+touch run-sanity-check/generate-all-started
```

Comment:
Instead of creating these marker files by using `touch`, can we just rely on actual files with content that are generated as part of the process?

Reply:
Removed - added check contents of library list and generated autoconfs